addons/l10n_ma/data/account_tax_data.xml

```diff
@@ -11,40 +11,728 @@
     </record>
 
     <!-- Account Tags -->
-    <record id="tax_tag_01" model="account.account.tag">
-        <field name="name">TVA 20% VENTES</field>
+    <record id="tag_ma_01" model="account.account.tag">
+        <field name="name">Ventilation du chiffre d’affaires total</field>
         <field name="applicability">taxes</field>
     </record>
-    <record id="tax_tag_02" model="account.account.tag">
-        <field name="name">TVA 14% VENTES</field>
+
+    <record id="tag_ma_02" model="account.account.tag">
+        <field name="name">Montant du chiffre d’affaires réalisé y compris les affaires non imposables (HT)</field>
+        <field name="applicability">taxes</field>
+    </record>
+
+    <record id="tag_ma_03" model="account.account.tag">
+        <field name="name">-Opérations situées hors champ d’application de la TVA</field>
+        <field name="applicability">taxes</field>
+    </record>
+
+    <record id="tag_ma_04" model="account.account.tag">
+        <field name="name">-Opérations exonérées sans droit à déduction (art 91du CGI)</field>
+        <field name="applicability">taxes</field>
+    </record>
+
+    <record id="tag_ma_05" model="account.account.tag">
+        <field name="name">-Opérations exonérées avec droit à déduction (art 92 du CGI)</field>
+        <field name="applicability">taxes</field>
+    </record>
+
+    <record id="tag_ma_06" model="account.account.tag">
+        <field name="name">-Opérations faites en suspension de la TVA (art 94 du CGI)</field>
+        <field name="applicability">taxes</field>
+    </record>
+
+    <record id="tag_ma_07" model="account.account.tag">
+        <field name="name">Chiffre d’affaires imposable à répartir (différence : ligne 10 – (20 + 30 +40 +50) (HT)</field>
+        <field name="applicability">taxes</field>
+    </record>
+
+    <record id="tag_ma_08" model="account.account.tag">
+        <field name="name">Ventilation du chiffre d’affaires imposable</field>
+        <field name="applicability">taxes</field>
+    </record>
+
```

Comment:
Defining tags for report lines that do not use them makes no sense. Please remove them ;)